pkg/propertyprovider/azure/provider_integration_test.go

```diff
@@ -725,39 +733,51 @@ var _ = Describe("azure property provider", func() {
 	Context("add a new node", Serial, Ordered, func() {
 		BeforeAll(shouldCreateNodes(nodes[0]))
 
-		It("should report correct properties", shouldReportCorrectPropertiesForNodes(nodes[0:1], nil))
+		It("should report correct properties", func() {
```

Comment:
Note: this encapsulation is needed so that Ginkgo tree building will not pass nil values to the function.